app/models/item.rb

```diff
@@ -4,8 +4,18 @@ class Item < ActiveRecord::Base
   validates :price, presence: true, numericality: { greater_than: 0 }
   validates :status, presence: true
 
-  has_attached_file :image, :default_url => "/Fat_unicorn.jpg"
-  validates_attachment_content_type :image, :content_type => ["image/jpg", "image/jpeg", "image/png"]
+  has_attached_file :image, styles: {
+      thumb: '100x100>',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.